snippets/base/templates/base/includes/snippet_as.css

```diff
@@ -36,8 +42,7 @@
     height: 12px;
     width: 12px;
     padding: 0;
-    background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgaGVpZ2h0PSIxNiIgdmlld0JveD0iMCAwIDE2IDE2Ij48cGF0aCBmaWxsPSJjb250ZXh0LWZpbGwiIGQ9Ik05LjQxNCA4bDMuNTMxLTMuNTMxYTEgMSAwIDEgMC0xLjQxNC0xLjQxNEw4IDYuNTg2IDQuNDY5IDMuMDU1YTEgMSAwIDEgMC0xLjQxNCAxLjQxNEw2LjU4NiA4bC0zLjUzMSAzLjUzMWExIDEgMCAxIDAgMS40MTQgMS40MTRMOCA5LjQxNGwzLjUzMSAzLjUzMWExIDEgMCAxIDAgMS40MTQtMS40MTR6Ii8+PC9zdmc+");
-    opacity: 0.6;
+    background-image: url("resource://activity-stream/data/content/assets/glyph-dismiss-16.svg");
```

Comment:
for some reason, when using the data-uri, the fill wasn't respected. So I pointed it straight to our version of the icon. Is that cool?

Reply:
I'll leave it up to @glogiotatidis but i'd say this probably isn't a good idea since this has to run in a preview and there's not really a great way to create fallback for this;
Maybe you could add a `.dark-theme #snippets {` block with a different data uri instead?

Reply:
> for some reason, when using the data-uri, the fill wasn't respected. So I pointed it straight to our version of the icon. Is that cool?

Unfortunately no. Let's keep the base64 image here.